src/app/src/sensorUtils.js

```diff
@@ -17,46 +29,86 @@ export function makeRiverGaugeRequest(id, isApiRequest) {
     return axios.get(url);
 }
 
-function parseCsvString(csvString) {
-    const data = Papa.parse(csvString, { header: true, comments: "#", dynamicTyping: true, skipEmptyLines: true });
-    return data;
+export function parseCsvString(csvString) {
+    const data = Papa.parse(csvString, {
+        header: true,
+        comments: '#',
+        dynamicTyping: true,
+        skipEmptyLines: true,
+    });
+
+    return data.data;
```

Comment:
I'm surprised this works because the previous destructuring was `data.value.timeSeries.data`

Reply:
Just kidding, I see you pass `response.data.value.timeSeries` from the action

Reply:
Oh yeah, I did that to make it easier to test in that it was easier to just pass that one piece of data.